src/main/java/org/folio/inventory/client/util/ClientWrapperUtil.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.inventory.client.util;
+
+import io.vertx.core.buffer.Buffer;
+import io.vertx.ext.web.client.HttpRequest;
+import org.folio.rest.tools.ClientHelpers;
+
+import static org.folio.inventory.dataimport.handlers.matching.util.EventHandlingUtil.OKAPI_TENANT;
+import static org.folio.inventory.dataimport.handlers.matching.util.EventHandlingUtil.OKAPI_TOKEN;
+import static org.folio.inventory.dataimport.handlers.matching.util.EventHandlingUtil.OKAPI_URL;
+import static org.folio.inventory.dataimport.handlers.matching.util.EventHandlingUtil.OKAPI_USER_ID;
+
+public final class ClientWrapperUtil {
```

Comment:
@RomanChernetskyi  Please add documentation for this new Util. The main purpose of it.